script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -199,12 +199,15 @@ The girls stare at them coldly with a gaze that says 'boys are pathetic'.
 // (in bed, but apparently also in general). I don't have, and there may not be,
 // anything to keep the reference (and while マグロ女 is pretty famous,
 // ドビウオ女 seems to be really obscure honestly).
-"I love the eager to %{i}please%{/i} reaction, thanks guys!
+// Okay, トビウオ also comes up way later in Ciel route (10_01_CIEL10_5_2, 28196), so should
+// probably reify it here so that people can make the connection.
+// We have to spirit a fair bit of prose, but still works in context.
+"Ooh, you're so eager to please, like flying fish leaping right into the boat!
```

Comment:
Is トビウオ a particularly common fish to mention? It's feels weirdly specific enough to me that it could plausibly be intentional. Or does Nasu just like flying fish? I guess the real thing that would make me vote connection is if his demise was something to do with him seeking/throwing himself at Noel, but instead she just kinda happens to him.

Reply:
It's just an easy comparison. Most people know about flying fish in Japan, and they obviously seem pretty eager.